src/stream-chat-net/IClient.cs

```diff
@@ -7,6 +7,7 @@ namespace StreamChat
     public interface IClient
     {
         IUsers Users { get; }
+        IPermissionService Permissions { get; }
```

Comment:
I want to rename IUsers to IUserService in a future refactor.

Reply:
We don't need to be same between SDKs but the most idiomatic in the language ecosystem/namings

Reply:
Maybe it's better to add `Service` in one go. Otherwise permission looks different than others

Reply:
good point

Reply:
reverted